src/LLMService.py

```diff
@@ -1,4 +1,6 @@
 import os
+from pprint import pprint
```

Comment:
Can you also adjust other places where we simply used `print("".center(80, '-'))`?